build.psm1

```diff
@@ -537,7 +537,8 @@ Fix steps:
     # publish powershell.config.json
     $config = @{}
     if ($environment.IsWindows) {
-        $config = @{ "Microsoft.PowerShell:ExecutionPolicy" = "RemoteSigned" }
+        $config = @{ "Microsoft.PowerShell:ExecutionPolicy" = "RemoteSigned";
+                     "WindowsPowerShellCompatibilityModuleDenyList" = @("PSScheduledJob","BestPractices") }
```

Comment:
It seems @PaulHigin mentioned some other modules that do not work with remoting and serialization.

Reply:
Talked to Paul offline; he was referring to `PSScheduledJob`.

Reply:
The WSUS module should be added to the default list.

Reply:
@doctordns `UpdateServices` is the module name, right?

Reply:
correct

Reply:
correct

Reply:
Tried it and added `UpdateServices` to default list.